src/index.js

```diff
@@ -0,0 +1,84 @@
+const fs = () => window[window._fs_namespace];
+let didInit = false;
```

Comment:
I don't think it's necessary to have this as a module-level variable. Mutable references defined in module scope always give me pause because it's sometimes hard to track where they are set and accessed.

This one is only used in one function, so it should be easy to pull this functionality into a small function like [_.once](https://lodash.com/docs/4.17.15#once). The implementation would only be a few lines:
```js
const once = fn => {
  let called = false

  return (...args) => {
    if (called) return
    fn(...args)
    called = true
  }
}
```

Then, when exporting the map of functions, we would export `init` like this: `init: once(init)`. We could even have `once` take a second function argument that executes on subsequent calls so we can still log the warning.

Reply:
I'm down with this.  I'd still want to console.warn(`you've already called ${fn.name}`), or something like that, so there'd be a side-effect added to the `if(called)` condition.